### ISSUE TITLE
Allow multiple domains being added to SAN via tlsextradomain(s)

### ISSUE THREAD

**Issue by stp-ip** [lightningnetwork/lnd#1492]
### Background

tlsextradomain is only usable to add a single domain to the cert.

### Your environment

* version of `lnd`
https://github.com/lightningnetwork/lnd/commit/b8fecfca71507643e2ba3cc32b65c9de87790597
* which operating system (`uname -a` on *Nix)
 4.15.0-1-ARCH (Archlinux)
* version of `btcd`, `bitcoind`, or other backend
bitcoind
* any other relevant environment details

### Steps to reproduce

Add 2 tlsextradomain entries.

### Expected behaviour

Use all tlsextradomain entries (same probably for tlsextraip I assume)

### Actual behaviour

Only last one will be used.

**Comment by Roasbeef** [comment]
If you look at the config (and also the command line help), it only accepts a single domain. The name of the parameter is `tlsextradomain` which implies a singular domain. Can you rephrase the title of this issue and also the body to denote that this is a _feature request_? Thanks!

**Comment by stp-ip** [comment]
Yeah I looked at the code, but having only one domain (and ip) addable seemed imperfect.

Will change the title.
